mlflow/server/js/src/experiment-tracking/components/experiment-page/components/header/ExperimentViewHeader.tsx

```diff
@@ -206,6 +218,15 @@ export const ExperimentViewHeader = React.memo(
           />
           {getShareButton()}
         </div>
+        <RenameExperimentModal
+          experimentId={experiment.experimentId}
+          experimentName={experiment.name}
+          isOpen={showRenameExperimentModal}
+          onClose={() => setShowRenameExperimentModal(false)}
+          onExperimentRenamed={() => {
+            invalidateExperimentList();
+          }}
```

Comment:
nit: redundant thunk
```suggestion
          onExperimentRenamed={invalidateExperimentList}
```